recipes/act-atmos/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "act-atmos" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7cf956c516c736f0d5ea50fd81e41f88de3d6ce40803ef05cbd61b4fc8b541d
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Can you remove this line and add `noarch: python`?

Reply:
Hi @scopatz, I tried that below, but it was having a conflict on windows with the dependencies associated with metpy.  We need pyproj 2.3.1 for the CRS function, but that was conflicting with metpy for some reason.  Thoughts?

Reply:
If it fails to install on windows, that is fine.  It is better to have a noarch package that you know won't work on all platforms

Reply:
Okay, resubmitted with noarch: python.  I'll expect the staged_recipes to fail for windows.  Will this still be able to get accepted?  Thanks for your help on this!